### ISSUE TITLE
Reformat default help display

### ISSUE THREAD

**Issue by dvdsgl** [Microsoft/mobile-center-cli#169]
Here's the current output of running `mobile-center` without any arguments. We should spruce it up!

```
$ mobile-center  
mobile-center 


    mobile-center
    Command to start with the Mobile Center CLI


    mobile-center apps           Commands to manage applications in Mobile Center                                                                
                                                                                                                                                
    mobile-center build          Commands for queueing builds, getting build status and downloading binaries and logs for completed builds       
                                                                                                                                                
    mobile-center crashes        Command to upload symbols to get symbolicated crash reports on Mobile Center                                    
                                                                                                                                                
    mobile-center distribute     Commands for the releases and distribution groups manipulation                                                  
                                                                                                                                                
    mobile-center profile        Commands to manage profile information in Mobile Center                                                         
                                                                                                                                                
    mobile-center telemetry      Commands to manage the sending of telemetry from Mobile Center commands                                         
                                                                                                                                                
    mobile-center test           Command for preparing, starting and checking state of Mobile Center tests                                       
                                                                                                                                                
    mobile-center tokens         Commands to manage API tokens in Mobile Center                                                                  
                                                                                                                                                
    mobile-center help           Get command or category help                                                                                    
                                                                                                                                                
    mobile-center login          Login to Mobile Center                                                                                          
                                                                                                                                                
    mobile-center logout         Logout from Mobile Center                                                                                       
```

**Comment by dvdsgl** [comment]
I suspect that these outputs are largely driven by our modular command architecture, but here are a few modifications I'd like to explore:

* The first output of the `mobile-center` command is "mobile-center\n\n\n\tmobile-center", so I've removed that.
* I simplified the copy to be more friendly and less verbose
* I use a secondary foreground color to create some contrast between commands and explanations. Perhaps there is an `npm` module that can help us with colorschemes? I am not familiar with coloring text in terminals, and how we could determine a foreground color with less contrast for this purpose, but I've seen other tools do it so it must be possible.
* Following the example of `git` and `npm`, I decided not to repeat `mobile-center` before every command explanation.
* I treat `help` as a special command, rendered at the end with further instruction.

<img width="1154" alt="mobile-center 2x" src="https://cloud.githubusercontent.com/assets/108197/25295447/5d64faf0-2698-11e7-8724-c3ad1e13d6ba.png">
